composer.json

```diff
@@ -13,26 +13,38 @@
     "prefer-stable": true,
     "require": {
         "php": ">=8.1",
-        "doctrine/doctrine-bundle": "^2.10.0",
         "makinacorpus/query-builder": "^1.6.1",
+        "psr/log": "^3.0",
         "symfony/config": "^6.0|^7.0",
-        "symfony/console": "^6.0|^7.0",
-        "symfony/dependency-injection": "^6.0|^7.0",
         "symfony/filesystem": "^6.0|^7.0",
         "symfony/finder": "^6.0|^7.0",
         "symfony/options-resolver": "^6.0|^7.0",
-        "symfony/password-hasher": "^6.0|^7.0",
         "symfony/process": "^6.0|^7.0",
         "symfony/yaml": "^6.0|^7.0"
     },
     "require-dev": {
+        "doctrine/doctrine-bundle": "^2.10.0",
         "doctrine/orm": "^2.15|^3.0",
         "friendsofphp/php-cs-fixer": "^3.34",
         "phpstan/phpstan": "^1.10",
         "phpunit/phpunit": "^10.4",
+        "symfony/console": "^6.0|^7.0",
```

Comment:
This is not only necessary on dev, no?

Reply:
Sorry I misread, I thought it was about the doctrine bundle...

Reply:
OK, what I wanted to do is:

* remove `symfony/console` from mandatory requirements in order to have the less hardwired dependencies possible,
* move it do dev dependencies for testing,
* console component is required for Symfony and the standalone build
* console component may conflict with an hypothetical future Laravel command, since it's own symfony/console version requirement may differ from ours.

@SimonMellerin that's why I want to keep it optional, so that potential other integrations can use, not use it, or use it in a different version.

Reply:
I don't know, as all current uses requires the Symfony/Console component, I find it odd to not require it.

May be we could address this hypothetical issue when it shows up ? 

Cause if I understand correctly, what you suggest imply that if one wants to use the DbToolsBundle on a random composer project, he should enter `composer require makinacorpus/db-tools-bundle symfony/console`, that's it ?

If so, it's a shame

Reply:
True, let's keep it mandatory for now, but we will have to think thoroughly about this when we'll diversify integrations. We will eventually hit a hard barrier were some X or Y framework will cause dependency conflicts.